File: public/docs/_examples/intro/dart/lib/logger_service.dart
// #docregion
library developer_guide_intro.logger_service;

import 'dart:html';

import 'package:angular2/angular2.dart';

@Injectable()
class Logger {
  void log(Object msg) => window.console.log(msg);

  void error(Object msg) => window.console.error(msg);

  void warn(Object msg) => window.console.warn(msg);
}

File: public/docs/_examples/cb-dependency-injection/ts/app/logger.service.ts
// #docregion
export class LoggerService{
  
  logInfo(msg:string){
    console.log(`INFO: ${msg}`);
  }
  
  logDebug(msg:string){
    console.log(`DEBUG: ${msg}`);
  }
  
  logError(msg:string){
    console.log(`ERROR: ${msg}`);
  }
}